plan_delegator/src/plan_delegator.cpp

```diff
@@ -35,9 +35,9 @@ namespace plan_delegator
         traj_pub_ = nh_.advertise<cav_msgs::TrajectoryPlan>("plan_trajectory", 5);
         plan_sub_ = nh_.subscribe("final_maneuver_plan", 5, &PlanDelegator::maneuverPlanCallback, this);
         twist_sub_ = nh_.subscribe<geometry_msgs::TwistStamped>("current_velocity", 5,
-            [&](const geometry_msgs::TwistStampedConstPtr& twist) {latest_twist_ = *twist;});
+            [&latest_twist_ = this->latest_twist_](const geometry_msgs::TwistStampedConstPtr& twist) {latest_twist_ = *twist;});
```

Comment:
This change has me concerned. 

I have never seen assignment inside the capture list of a c++ lambda. 
Same for below. What was the error that prompted this change?

Reply:
The issue is [here](https://sonarcloud.io/project/issues?id=usdot-fhwa-stol_CARMAPlatform&open=AW7IuXsy11XIXiA9YnCW&resolved=false&severities=CRITICAL) I wasn't sure about this either. just havign `[&latest_twist_]` gave me:
error: capture of non-variable 
error: ‘this’ was not captured for this lambda function
error: invalid use of non-static data member

Reply:
When you do a capture on a member variable you need to capture this instead. So the proper syntax would be
```
[this] () {
this->my_variable_ = val;
}
```